### DISCUSSION TITLE
Cannot find module '@openpgp/web-stream-tools'

### DISCUSSION THREAD

**Discussion by berthoug** [openpgpjs/openpgpjs#1673] [Q&A] [answered]
I'm not sure if it is a bug or a problem on my side so Starting with a question here.

I have a project depending on version 5.9 of the library and when I try to run transpile on it I'm getting
```

node_modules/openpgp/openpgp.d.ts:10:85 - error TS2307: Cannot find module '@openpgp/web-stream-tools' or its corresponding type declarations.

10 import type { WebStream as GenericWebStream, NodeStream as GenericNodeStream } from '@openpgp/web-stream-tools';

```

I can see in the openpgpjs' [package.json](https://github.com/openpgpjs/openpgpjs/blob/main/package.json) that `@openpgp/web-stream-tools` is a dev dependency while it is imported directly in the openpgp.js file, which does not seem to be used only for local development. Any reason why it is not a dependency?

And what should I do to be able to transpile my code?

**Comment by larabr** [comment, marked as answer]
See https://github.com/openpgpjs/openpgpjs#typescript

**Reply by berthoug** [reply]
Thanks.
And sorry for missing it.
